lib/streamy/event.rb

```diff
@@ -26,6 +26,12 @@ def publish
       )
     end
 
+    def self.deliver
+      priority :manual
```

Comment:
```suggestion
      priority :batched
```

I think we can re-use the `batched` sending here, which will protect us from very large batches

Reply:
Ok, that makes sense 👍

Reply:
This would also solve my first question

> Currently there is also a batched mode which calls deliver_messages when the buffer size is reached. However, with this implementation, when calling the deliver block all produced messages are getting published. This could cause some unexpected side effects when using the batch mode. I don't know an easy way to get around that. IIRC the batch mode is currently not used so maybe we can just drop it to avoid confusion?

👍